plugins/ssrsearch/g_search.mlg

```diff
@@ -299,6 +299,8 @@ let ssrdisplaysearch gr env t =
   let pr_res = pr_global gr ++ str ":" ++ spc () ++ pr_lconstr_env env Evd.empty t in
   Feedback.msg_notice (hov 2 pr_res ++ fnl ())
 
+let _ = CWarnings.set_flags "-ssr-search-moved"
```

Comment:
This is just silencing the warning, but not for people who use `Set Warning "+all"`. What about the following solution instead: ssreflect defines a reference that is set to `false`, the warning checks the value of the reference and is only triggered if it is `false`, ssrsearch sets the value of the reference to `true`.

Reply:
I actually thought of that, but thought it would be overkill, added the ref.